features/305.Big_Data_Spark.feature

```diff
@@ -0,0 +1,133 @@
+Feature: test tutorial 305.Big Data (Spark)
+
+  This is the feature file of the FIWARE Step by Step tutorial for Big Data (Spark)
+  url: https://fiware-tutorials.readthedocs.io/en/latest/big-data-spark.html
+  git-clone: https://github.com/FIWARE/tutorials.Big-Data-Spark.git
+  git-directory: /tmp/tutorials.Big-Data-Spark
+  shell-commands: git checkout NGSI-v2 ; ./services create; ./services start
+  clean-shell-commands: ./services stop
+
+
+    Background:
+        Given I set the tutorial 305.Spark
+
+    # COMPILING A JAR FILE FOR SPARK --
+    Scenario: Compiling a Jar File For Spark
+        Given I prepare the script create-spark-jar.sh to be run
+        And   I set the environ variable WORKING_DIR to "cosmos-examples" under git
+        And   I set the environ variable OUTPUT_DIR to "cosmos-examples/target" under git
+        When  I exec the script as in the tutorial page
+        Then  I expect the scripts shows a result of 0
+
+    #   LOGGER - READING CONTEXT DATA STREAMS
+    # - Logger - Installing the JAR
+    Scenario: Run the generated jar in Spark. This process won't end
+        Given I prepare the script run-spark-jar.sh to be run
+        When  I open a new shell terminal spark-run-jar and exec "features/data/305.Big_Data_Spark/run-spark-jar.sh"
```

Comment:
When  I open a new shell terminal spark-run-jar and execute "features/data/305.Big_Data_Spark/run-spark-jar.sh"

Reply:
To be consistently, it should be run and not exec

Reply:
idem

Reply:
NTC